django/db/backends/base/schema.py

```diff
@@ -1063,9 +1063,25 @@ def _field_should_be_altered(self, old_field, new_field):
         _, new_path, new_args, new_kwargs = new_field.deconstruct()
```

Comment:
looks like we explicitly discard `name` so that's why it doesn't need to be present in `non_database_fields`.

Reply:
Yes, it's also already tested in `tests.migrations/test_operations.OperationTests.test_rename_field_with_db_column`.